.eleventy.js

```diff
@@ -292,9 +292,12 @@ module.exports = function(eleventyConfig) {
 
     eleventyConfig.addFilter("relatedPosts", function (collection = []) {
         const { tags: requiredTags, page } = this.ctx;
-        return collection.filter(post => {
-            return post.url !== page.url && requiredTags?.every(tag => post.data.tags?.includes(tag));
-        });
+        return collection
+            .map(post => {
+                const commonTags = requiredTags?.reduce((count, tag) => count + (post.data.tags?.includes(tag) ? 1 : 0), 0);
+                return { ...post, commonTags };
+            })
+            .filter(post => post.url !== page.url && post.commonTags >= requiredTags.length - 1);
```

Comment:
@ZJvandeWeg I'm having second thought about this change, because I saw you worked on the tags for the UNS articles, and right now it looks like this:
<img width="319" alt="Screenshot 2024-01-18 at 16 34 40" src="https://github.com/FlowFuse/website/assets/129537638/681bcf6b-8ff6-4ff6-9e12-8cb916f403ac">

But with this change, there are too many newer related articles, and those ones ⬆️ are excluded from the list:
<img width="313" alt="Screenshot 2024-01-18 at 16 34 57" src="https://github.com/FlowFuse/website/assets/129537638/2fbe4f61-3006-45bf-a36b-584bd01c0c91">

So, I'm thinking of keeping it as we have it now, and only adding the "else" in the post.njk so it's never an empty section. WDYT?